xbmc/platform/darwin/ios-common/DarwinEmbedNowPlayingInfoManager.mm

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright (C) 2010-2019 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#import "DarwinEmbedNowPlayingInfoManager.h"
+
+#import "platform/darwin/ios/XBMCController.h"
+
+#import <Foundation/Foundation.h>
+#import <MediaPlayer/MPMediaItem.h>
+#import <MediaPlayer/MPNowPlayingInfoCenter.h>
+
+@implementation DarwinEmbedNowPlayingInfoManager
+
+@synthesize playbackState;
+
+#pragma mark - Now Playing routines
+
+- (void)setDarwinEmbedNowPlayingInfo:(NSDictionary*)info
+{
+  self.nowPlayingInfo = info;
+  [MPNowPlayingInfoCenter defaultCenter].nowPlayingInfo = self.nowPlayingInfo;
+}
+
+- (void)onPlay:(NSDictionary*)item
+{
+  NSMutableDictionary* dict = [[NSMutableDictionary alloc] init];
```

Comment:
`[[SomeClass alloc] init]` is synonymous to `[SomeClass new]`: the latter is shorter, but it's probably just a personal preference

Reply:
Happy to go the shorter one. Thats the way ive been doing the tvos stuff anyway.